server/src/test/java/io/crate/expression/scalar/arithmetic/ExpFunctionTest.java

```diff
@@ -33,7 +33,17 @@ public class ExpFunctionTest extends ScalarTestCase {
     public void test_exp_scalar() {
         assertNormalize("exp(1)", isLiteral(2));
         assertNormalize("exp(1::bigint)", isLiteral(2L));
-        assertNormalize("exp(1.0)", isLiteral(2.718281828459045));
+        if (isAppleSilicon()) {
+            assertNormalize("exp(1.0)", isLiteral(2.7182818284590455));
+        } else {
+            assertNormalize("exp(1.0)", isLiteral(2.718281828459045));
+        }
```

Comment:
Maybe we could have a way to account for floating point in-accuracies like assertj matchers would support? Than we don't need to distinguish here either and can remove the `isAppleSilicon`